roles/bootstrap-os/tasks/main.yml

```diff
@@ -1,4 +1,26 @@
 ---
+- name: Fetch /etc/os-release
+  raw: cat /etc/os-release
```

Comment:
reading at ansible code 
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/facts/system/lsb.py and https://github.com/ansible/ansible/blob/ab96bbdef9a1a42b7cf3cb08db8056994fa3aaf9/lib/ansible/module_utils/facts/system/distribution.py

it seems there's more cases depending of the OS. 
But if it works for all the os we support now, that's fine

Reply:
The idea was to support the OS versions we declared inside README.md